.github/workflows/first-time-setup.yaml

```diff
@@ -70,8 +70,8 @@ jobs:
         run: |
           user="${{ github.repository_owner }}"
           description="An engaging 1-3 sentence description of your lab."
-          printf "USER=${user}" >> $GITHUB_ENV
-          printf "DESCRIPTION=${description}" >> $GITHUB_ENV
+          printf "\nUSER=${user}" >> $GITHUB_ENV
```

Comment:
`$GITHUB_ENV` seems to act like a regular `.env` file. Forgot to add new lines here, so these vars were getting messed up like:

```
OTHER_VAR=other value
USER=some-usernameDESCRIPTION=description of lab
```

Reply:
I think I read that printf has slightly better cross-platform compatibility so I guess I just did that instead of echo with the `-e` flag.